genai/video_generation/videogen_with_txt.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def generate_videos(output_gcs_uri: str) -> str:
+    # [START googlegenaisdk_videogen_with_txt]
+    import time
+    from google import genai
+    from google.genai.types import GenerateVideosConfig
+
+    client = genai.Client()
+    prompt = "a cat reading a book"
+
+    # TODO(developer): Update and un-comment below line
+    # output_gcs_uri = "gs://your-bucket/your-prefix"
+
+    operation = client.models.generate_videos(
+        model="veo-2.0-generate-001",
+        prompt=prompt,
```

Comment:
We can actually skip this `prompt` variable def

```suggestion
        prompt="a cat reading a book",
```